team/482451/montyhall.py

```diff
@@ -0,0 +1,30 @@
+from random import randint
+
+c = 0
+list2 = []
+
+while c != 100000:
+
+    p = randint(0, 2)
+    list = [0, 0, 0]
+
+    if p == 0:
+        list[0] = 1
+    elif p == 1:
+        list[1] = 1
+    else:
+        list[2] = 1
+
+    selection = randint(0, 2)
+    list.pop(selection)
+
+    if list[0] == 0 and list[1] == 0:
```

Comment:
Redundant use of if-else. Simply check for the presence of a number 1 in any position.

Reply:
Depending on the programmer level, we can consider showing the solution if not trivial:

`list2.append(any(list))`